public/views/backup.html

```diff
@@ -49,30 +49,30 @@ <h5 class="text-center" translate>Write your wallet seed</h5>
             To restore this {{index.m}}-{{index.n}} <b>shared</b> wallet you will need
           </span>:
           <ol class="m10t columns size-14 text-gray">
-            <li translate>Your wallet seed and access to the server that coordinated the initial wallet creation. You still need {{index.m}} keys to spend.</li>
-            <li translate><b>OR</b> the wallet seeds of <b>all</b> copayers in the wallet</li>
+            <li translate>Your wallet recovery phrase and access to the server that coordinated the initial wallet creation. You still need {{index.m}} keys to spend.</li>
+            <li translate><b>OR</b> the wallet recovery phrases of <b>all</b> copayers in the wallet</li>
           </ol>
           </span>
         </div>
       </div>
 
       <div class="row m20t" ng-show="wordsC.deleted">
         <div class="columns size-14 text-gray text-center" translate>
-          Wallet seed not available. You can still export it from Advanced &gt; Export.
+          Recovery phrase not available. You can still export it from Advanced &gt; Export.
         </div>
       </div>
 
       <div ng-show="wordsC.mnemonicWords || (wordsC.credentialsEncrypted && !wordsC.deleted)">
         <p class="text-center columns text-gray"  ng-show="index.n==1 && wordsC.step == 1">
           <span translate>
-            You need the wallet seed to restore this personal wallet. Write it down and keep them somewhere safe.
+            You need the recovery phrase to restore this personal wallet. This set of words (also called "seed") fully contains your bitcoins by reconstructing the private key without any contact with a distant server (as specified in the BIP39 Standard). Write it down and keep them somewhere safe.
```

Comment:
> This set of words (also called "seed") fully contains your bitcoins by reconstructing the private key without any contact with a distant server (as specified in the BIP39 Standard)

Maybe keep it simple and direct:
"These 12 words are all you will ever need to gain access to your bitcoins."

IMO if you call it `seed` maybe you need to explain yourself but not the other way around as power users already know what we are talking about.